public/vendor/bootstrap/js/bootstrap.bundle.js

```diff
@@ -1,13 +1,17 @@
 /*!
```

Comment:
Might be better to avoid prettifying the `vendor/` files, since they are just bundled from bootstrap and are not meant to be read by humans.